code/modules/mob/living/carbon/metroid/metroid.dm

```diff
@@ -899,90 +890,6 @@ mob/living/carbon/slime/var/temperature_resistance = T0C+75
 	icon = 'icons/obj/chemical.dmi'
 	icon_state = "bottle17"
 
-	/*afterattack(obj/target, mob/user , flag)
-		if(istype(target, /obj/item/slime_extract))
-			if(target.enhanced == 1)
-				user << "<span class='warning'> This extract has already been enhanced!</span>"
-				return ..()
-			if(target.Uses == 0)
-				user << "<span class='warning'> You can't enhance a used extract!</span>"
-				return ..()
-			user <<"You apply the enhancer. It now has triple the amount of uses."
-			target.Uses = 3
-			target.enahnced = 1
-			del(src)*/
-
-////////Adamantine Golem stuff I dunno where else to put it
-

```

Comment:
Yeah, I would recommend leaving the golem equipment and removing their brutemod, it's supposed to basically function as riot armor for them. Shadowlings use something similar for their "equipment."